test/test_env.py

```diff
@@ -1047,15 +1009,16 @@ def test_env_base_reset_flag(batch_size, max_steps=3):
     assert (td["next", "done"] == 1).all()
     assert (td["next", "observation"] == max_steps + 1).all()
 
-    _reset = env.done_spec.rand()
+    env._allow_done_after_reset = True
```

Comment:
again this flag should not be used in the tests, this is for envs that will be done on the _reset dimensions.

Reply:
Removed